configs/train.yaml

```diff
@@ -3,13 +3,13 @@
 # specify here default training configuration
 defaults:
   - _self_
-  - datamodule: synthetic.yaml
-  - model: synthetic.yaml
+  - datamodule: cora.yaml
```

Comment:
why changed here?

Reply:
I use this dataset for experiment. To change the default values would be more convenient for me.

Reply:
Better not change default values to adapt your usage. You can override parameter in command line.